src/components/PersonalInfoForm/PersonalInfoForm.js

```diff
@@ -45,6 +57,16 @@ function PersonalInfoForm() {
           newMessage = "Please enter a valid email!";
         }
         break;
+      // Add validation rules for GitHub, website, and LinkedIn here
+      case "github":
+        // Add GitHub validation logic here
+        break;
+      case "website":
+        // Add website validation logic here
+        break;
+      case "linkedin":
+        // Add LinkedIn validation logic here
+        break;
```

Comment:
The validation is not implemented. If you are not planning on implementing it here, it would be great to open an issue to track this!

Reply:
@fissoreg Fixed